### DISCUSSION TITLE
v3.2.0 Beta | Beta releases | AUR, Snap and Flatpaks

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#88] [Announcements]
# Beta Release

Hi everyone, this project has been a somewhat busy project therefore I decided to adapt a bit the release process. 

I am happy to announce [v3.2.0 is now Beta release](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v3.2.0-beta.1)

From now on will be releasing beta versions instead of development versions/releases. What does a beta release exactly mean? 

This means that version should be somewhat stable and that there will be no more features being added to that particular version with only related bug-fixes may go in!

Development branch will now be a pure development branch and the branch taking in all the pull requests. 

# AUR Packages 

With this in mind there is now one new arch AUR package `streamdeck-ui-beta` dedicated for beta releases. 

The previous `streamdeck-ui-develop` has now changed to install from development branch and its not tied to any particular release. This package should be considered unstable and you should only use if if you want to test the stuff that is currently under development without having  necessarily to deal with all the overhead involved with the development setup. 

# Snap and Flatpacks 

Python is not really friendly when managing dependencies and installing the streamdeck-ui has proven to be quite challenging for some users, to solve this I intend to start working on snap and flatpack packages over the next few weeks and I will provide more news when I have them. just be a bit patient with me as my time is somewhat limited. 

Please, please, I  am still writing this and I can already hear you so before you start shredding your keyboard, I kindly ask you to please refrain from adding comments "when \<insert your preferd packaging system here\>? It is so much better". 

 am particularly opinionated about the Linux packaging ecosystem and for now I will be providing those two. Later? who knows, if you really what it please be my guest and make your pull request, I will gladly take a look at it and accept it if all looks good!

**Comment by coolapso** [comment]
oooh and where are my manners? 

# THANK YOU TO ALL THE CONTRIBUTORS!

**Comment by oparada1988** [comment]
Thank you for your hard work! I, and many other user appreciate your hard work!
